hedera-mirror-rest/topicmessage.js

```diff
@@ -400,6 +409,39 @@ const getMessages = async (pgSqlQuery, pgSqlParams, preQueryHint) => {
   return rows.map((row) => new TopicMessage(row));
 };
 
+const getQueryWithTopicMessageLookupTimestamps = async (topicId, seqNum) => {
+  // Get the timestamp range from the topic message lookup table
+  const sequenceNumber = BigInt(seqNum);
```

Comment:
unnecessary since you have an explicit type conversion in the SQL

```suggestion
  const sequenceNumber = BigInt(seqNum);
```

Reply:
Thanks I have updated this.